src/sentry/static/sentry/app/components/organizations/headerItem.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+class HeaderItem extends React.Component {
+  static propTypes = {
+    label: PropTypes.node,
+    align: PropTypes.oneOf(['right', 'left']),
+
+    /**
+     * className for <Label> component
+     */
+    labelClassName: PropTypes.string,
+  };
+
+  static defaultProps = {
+    align: 'right',
+  };
+
+  render() {
+    const {className, labelClassName, label, align, children} = this.props;
+
+    return (
+      <StyledHeaderItem align={align} className={className}>
```

Comment:
I wanted to leave this the way it was, but also wanted `align` to have a default property, so had to refactor this a bit.